[Fix] com.ncapdevi:frag-nav dependency not found

juaniunzu opened the pull request:
This PR removes the old `jcenter()` repository and replaces it with the jitpack one. Also, the implementation of the dependency is changed from `com.ncapdevi:frag-nav` to `com.github.ncapdevi:fragnav`.

This solution is discussed [here](https://github.com/ncapdevi/FragNav/issues/255).

This fixes [this issue](https://github.com/techyourchance/android-custom-views-course/issues/1).

grishaster80 commented:
Thank you for the fix!

I can confirm that it works. It was really frustrating to start a course and face an issue with dependency resolving. Hope that it will be merged by @techyourchance

techyourchance commented:
Wow, @juaniunzu , thank you for this contribution. And thanks @grishaster80 for bumping it up, as I didn't notice the notification.
Indeed, this is the required fix, and I've been applying it in my courses, client projects and personal projects for the past months. I'm surprised that no student raised a question about this issue before.
Unfortunately, I can't just accept this PR and call it a day. Since we develop this application gradually throughouhg the course, and each commit should be a working entry point into the app (i.e. I should be able to checkout any commit and build the app successfully), I'll need to perform an interactive rebase and change the git history of this project. It will be a breaking change for students who already cloned this repo, but I must do that to avoid confusion among future students. In these cases, I usually keep the old history in a standalone branch, in case the existing students will need it.
Once again, thank you both for your contributions!

techyourchance commented:
@juaniunzu , I rebased the git history in the master branch and moved the old code to the `old-master` branch. 
Now I've got an idea: I can actually merge your PR into `old-master`, to allow existing students pull the fix without cloning the project again. Unfortunately, I can't find how to change the target branch of this PR myself. Therefore, if you have a moment, could you please open a new PR into `old-master` branch (same contents as this PR) and we'll merge it?

juaniunzu commented:
> @juaniunzu , I rebased the git history in the master branch and moved the old code to the `old-master` branch. Now I've got an idea: I can actually merge your PR into `old-master`, to allow existing students pull the fix without cloning the project again. Unfortunately, I can't find how to change the target branch of this PR myself. Therefore, if you have a moment, could you please open a new PR into `old-master` branch (same contents as this PR) and we'll merge it?

Hi Vasiliy! Sure, I'll open a new PR.
Edit: [Done](https://github.com/techyourchance/android-custom-views-course/pull/3)!
